.github/workflows/check-full.yaml

```diff
@@ -19,10 +23,17 @@ jobs:
       matrix:
         config:
           - {os: macOS-latest,   r: 'release'}
+
           - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
+          # Use 3.6 to trigger usage of RTools35
+          - {os: windows-latest, r: '3.6'}
+
+          # Use older ubuntu to maximise backward compatibility
+          - {os: ubuntu-18.04,   r: 'devel', http-user-agent: 'release'}
```

Comment:
why did os changed fron ubuntu-latest to ubuntu-<version>?

Reply:
Not sure about that -- I just run `usethis::use_github_action("check-full")` to get the latest version of that workflow (from r-lib/actions).

Reply:
@MirijaHa please see if this answers your question and if you approve we can merge and get it out of the way.

Reply:
LGTM now, will approve.
Just a small future headache, as ubuntu version is now hardcoded we should make  even more sure to regularly update the gh actions workflow. Obviously one could simply set a monthly reminder to update it, but I wonder if you use a more informed system to make sure your workflows do not outdate?

Reply:
Yeah, maintaining the workflows is indeed a quite time consuming task. I guess we could automate the process to run `usethis::use_github_actions("check-full")`, here and maybe all CRAN packages where we make an effort to keep the package compliant with that that workflow expects.

Reply:
Maybe we could trigger the update only when a commit touches r-lib/actions/examples/check-full.yaml